scripts/main.js

```diff
@@ -1,5 +1,5 @@
 $.ajax({
-    url: 'http://namegame.willowtreemobile.com:2000',
+    url: 'https://willowtreeapps.com/api/v1.0/profiles/',
```

Comment:
yeah, i totally agree. I think there are a lot of possibilities opened by this API and contentful's query params. even their image endpoints are super smart.